docs/workflow/building/coreclr/nativeaot.md

```diff
@@ -43,7 +43,7 @@ The workflow looks like this:
 
 ## Running tests
 
-If you haven't built the tests yet, run `src\tests\build[.cmd|.sh] nativeaot [Debug|Release] tree nativeaot`. This will build the smoke tests only - they usually suffice to ensure the runtime and compiler is in a workable shape. To build all Pri-0 tests, drop the `tree nativeaot` parameter. The `Debug`/`Release` parameter should match the build configuration you used to build the runtime.
+If you haven't built the tests yet, run `src\tests\build[.cmd|.sh] nativeaot [Debug|Release] tree nativeaot`. This will build the smoke tests only - they usually suffice to ensure the runtime and compiler is in a workable shape. To build all Pri-0 tests, drop the `tree nativeaot` parameter. The `Debug`/`Release` parameter should match the build configuration you used to build the runtime. If you want build just subset of Pri-0 tests, you may use `src\tests\build[.cmd|.sh] nativeaot [Debug|Release] tree path_spec`, for example if I want build all tests under `src\tests\Interop\ICustomMarshallers` you can use `src\tests\build[.cmd|.sh] nativeaot [Debug|Release] tree Iterop\ICustomMarshallers`. You can notice, that `nativeaot` is just one of folders inside `src\tests`. You can build more then one subsets like that, before running tests, if neeeded. This is means that if you build two folders, `nativeaot` and `Interop\ICustomMarshallers` and after some time want to run just one, you have to manually go to `artifacts\tests\coreclr\[Windows|Linux|OSX[.x64.[Debug|Release]\` folder and remove built tests.
```

Comment:
This is starting to duplicate a good chunk of the general coreclr testing docs: 

https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/coreclr/windows-test-instructions.md#building-test-subsets

I think it would be better to add a link to the coreclr testing docs from here instead, and update the coreclr testing docs as needed.